src/routes/index.ts

```diff
@@ -0,0 +1,45 @@
+import express, { NextFunction, Request, Response, Router } from 'express';
+import ContactRoutes from './contact-requests';
+import { HTTPError } from '../utils/http-error';
+
+// For this API, it is necessary to enable multiple routes to get or send different data.
+const router: Router = express.Router();
+export default router;
+
+// If a GET request is made with no route specified, return 'Code:You API Demo'.
+router.get('/', (req: Request, res: Response) => {
+    res.send('Code:You API Demo');
+});
+
+router.use('/contact-requests', ContactRoutes);
```

Comment:
This allows us to organize the routers in a more duplicatable way. Now, any additional collections can supply their prefix and then inside the route for that collection - no need to deal with specific name strings.

Reply:
Good idea! It's also a good spot for something like a /ping endpoint.